src/styles/components/page-intro.scss

```diff
@@ -0,0 +1,26 @@
+@import "../settings";
+@import "../colours";
+
+.page-intro {
+  .intro-content {
+    -webkit-column-count: 2; /* Chrome, Safari, Opera */
```

Comment:
It's better to leave the browser specific features to CSS post-processors. If we haven't got one set up, it might be worth setting it up now, otherwise these browser specific styles need to be added/removed and generally maintained with the rest of code.

Reply:
I agree, but am not sure if our current SCSS/CSS loading processors are doing this. Could you make an GitHub issue plse?

Reply:
Yes, I will make an issue for it. We currently don't have any proper CSS post-processing in place.